frontend/src/elm/Pages/Prototype/FileSelectBox.elm

```diff
@@ -1,36 +1,72 @@
 module Pages.Prototype.FileSelectBox exposing (..)
 
-import Html exposing (Html, div, text, br, input)
-import Html.Attributes exposing (style, type_)
+import Html exposing (Html, div, text, br, input, button, form)
+import Html.Attributes exposing (style, type_, id, name, value)
+import Html.Events exposing (onClick, onSubmit)
+import Json.Decode as Json
+import Result exposing (Result(..))
+import Pages.Prototype.Ports as Ports
+import Pages.Prototype.Response as Response
```

Comment:
`Pages.Prototype.Response` bundles repetitive JSON response decoding in one file.